sceptre/config.py

```diff
@@ -231,7 +231,7 @@ def resolver_constructor_factory(node_class):
         self.add_yaml_constructors(
             resolvers_folder, Resolver, resolver_constructor_factory
         )
-        external_resolver_folder = os.path.join(os.getcwd(), "resolvers")
+        external_resolver_folder = os.path.join(str(self.sceptre_dir), "resolvers")
```

Comment:
Yep so (assuming it is the unicode thing), we check that the directory passed in is a `str`. In fact it should check to make sure it's a `basestring` to avoid encoding issues like this. 

https://github.com/cloudreach/sceptre/blob/master/sceptre/helpers.py#L221